apps/aecore/lib/aecore/structures/transaction.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Aecore.Structures.Transaction do
+
+  alias Aecore.Structures.DataTx
+  alias Aecore.Structures.Account
+  alias Aecore.Chain.ChainState
+
+  @typedoc "Arbitrary structure data of a transaction"
+  @type payload :: map()
+
+  @typedoc "Structure of a custom transaction"
+  @type tx_struct :: map()
+
+  @typedoc "Reason for the error"
+  @type reason :: String.t()
+
+  @typedoc "The public key of the account"
+  @type pub_key :: binary()
```

Comment:
`@type pub_key()` is used in multiple places with different names `@type pubkey()` sometimes. Move it to the Wallet worker.